icons/banknote-x.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "AnnaSasDev",
+    "joffx"
+  ],
+  "tags": [
+    "money",
+    "cash",
+    "payment",
+    "finance",
+    "error",
+    "failed",
+    "rejected",
+    "canceled",
+    "declined"
```

Please add tags from the original icon & tags covering use cases as suggested by @AnnaSasDev:
```suggestion
    "bill",
    "currency",
    "money",
    "payment",
    "funds",
    "transaction",
    "cash",
    "finance",
    "error",
    "failed",
    "rejected",
    "canceled",
    "declined",
    "lost",
    "delete",
    "remove"
```